BaseWeapon/src/main/java/com/github/sef24sp4/baseweapon/BaseProjectile.java

```diff
@@ -0,0 +1,29 @@
+package com.github.sef24sp4.baseweapon;
+
+import com.github.sef24sp4.common.entities.IAttackingEntity;
+import com.github.sef24sp4.common.entities.ICollidableEntity;
+import com.github.sef24sp4.common.interfaces.IEntityManager;
+import com.github.sef24sp4.common.metadata.GameElementType;
+import com.github.sef24sp4.common.projectile.CommonProjectile;
+
+public class BaseProjectile extends CommonProjectile implements ICollidableEntity, IAttackingEntity {
+	private final double damage = 1;
+	protected BaseProjectile(ICollidableEntity shooter) {
+		super(shooter);
+	}
+
+	@Override
+	public void collide(IEntityManager entityManager, ICollidableEntity otherEntity) {
+		if (otherEntity.getType() != GameElementType.PLAYER){
+			if (otherEntity.getType() != GameElementType.PROJECTILE){
```

Comment:
And please combine this into a single if statement using or operator.

Reply:
I think I would use an AND, instead of an OR since I want the projectile to remove other entities except the player and itself.

Reply:
It needs to be an OR operator because you want to avoid removing the projectile if it collides with the shooter **or** another projectile.